src/service/resources-impl.js

```diff
@@ -74,7 +75,7 @@ let ChangeSizeRequestDef;
 /**
  * @interface
  */
-class OwnersDef {
+class OwnersDef extends Owners {
```

Comment:
I assume this "extends" relationship is temporary, right? `OwnersDef` in this file will just be moved into owners-impl.js? 

A4A then will register and use its own implementation of the interface?

Reply:
Yes. The goal of this PR is layout the structure, so we can move the remaining methods from OwnersDef to Owners in parallel. 

a4a will have its own impls of those interfaces.